pkg/kubelet/kubelet.go

```diff
@@ -2735,14 +2738,23 @@ func (kl *Kubelet) setNodeStatusMachineInfo(node *api.Node) {
 	// TODO: Post NotReady if we cannot get MachineInfo from cAdvisor. This needs to start
 	// cAdvisor locally, e.g. for test-cmd.sh, and in integration test.
 	info, err := kl.GetCachedMachineInfo()
+	gpuCount := 0
+
+	for _, plugin := range kl.gpuPlugins {
+		currCount, err := plugin.Capacity()
```

Comment:
Check that the vendor is nvidia? This logic is slightly different than what you have [here](https://github.com/Hui-Zhi/kubernetes/blob/94a42a351c17c668774c80b88912316cca155d91/pkg/kubelet/dockertools/docker_manager.go#L552)